packages/component-library/src/components/plmg-separator/plmg-separator.scss

```diff
@@ -0,0 +1,42 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Common styles to all shadow-DOM components
+// Remove this section if your component does not use shadow DOM
+// -------------------------------------------------
+:host {
+  display: block;
+  height: inherit;
+  width: inherit;
+}
+
+:host([hidden]) {
+  display: none;
+}
+// ------------- END Common styles -----------------
+
+// Component style
+// -------------------------------------------------
+.plmg-separator-container {
+  display: flex;
+  width: inherit;
+  height: inherit;
+}
+
+.plmg-separator {
+  display: block;
+  border-radius: tokens.$plmg-border-radius-xl;
+  border-color: tokens.$plmg-border-neutral;
```

Comment:
The usage of `border-color` makes the component look bad.
It is a rounded rectangle with a blank background and a grey border. Instead it should be a grey rectangle.

![image](https://user-images.githubusercontent.com/55230735/166682943-59a96b65-3e81-4712-a2a8-ce7cdb8fbbf9.png)

Reply:
On the screenshot above, I've zoomed a lot to highlight this behavior

Reply:
I refactored this to remove the border and set border:none which resolves the problem. Now the hr element's color is set using background-color property.